python/raft-ann-bench/src/raft-ann-bench/run/algos-cpu.yaml

```diff
@@ -0,0 +1,30 @@
+faiss_gpu_ivf_flat:
```

Comment:
I wonder if instead of two files maybe we could add an option here to specify whether the executable runs on the GPU? I'm just thinking it could be easier to maintain.